setup.py

```diff
@@ -1,15 +1,27 @@
+import re
 import setuptools
 import sys
 
 
 REQUIREMENTS = [
-    "kombu>=3.0.12,<4",
-    "msgpack-python>=0.4.7,<0.5",
-    "requests>=2,<3",
-    "six>=1.10.0,<2",
-    "statsd>=2.1,<2.2",
+    "kombu>=4.5.0,<4.6",
+    "six>=1.14.0,<2",
+    "msgpack-python>=0.5.6,<0.6",
+    "statsd>=3.3.0,<4",
 ]
 
+# Regex matching version pattern
+# (3 numerical values separated by `.`, semver style, followed by an optional pre-release marker)
+version_pattern = re.compile(r"\d+\.\d+\.\d+([.-][\w_-]+)?")
+
+
+def get_version():
+    with open("CHANGELOG.md", "r") as fn:
+        while True:
```

Comment:
Not a major thing, but if there is no line that matches the regex, won't this loop forever?
Any reason not to iterate over the lines in the file instead?

Reply:
Good catch @sujaymansingh. Copy-Pasta gone ~wrong~ as usual

Reply:
28c5ba7